src/App.tsx

```diff
@@ -1,31 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const placeholder = 'Type search word';
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newTerm = e.target.value;
+
+    setSearchTerm(newTerm);
+  };
+
+  const filteredMovies = moviesFromServer.filter((movie) => {
+    if (searchTerm === '' || searchTerm === 'Default Value') {
```

Comment:
```suggestion
    if (!searchTerm || searchTerm === 'Default Value') {
```
what is `default value` here?